README.md

```diff
@@ -1,9 +1,67 @@
 # Polymer Linter
 
-Catch errors in your Polymer project before even running your code.
+Analyze your Polymer project for errors.
 
-## Work in progress
+## Usage
 
-Polymer Linter is being developed to replace [polylint][1], but is not yet complete.  For a working linter, use [polylint][1].
+The linter can be run in one of two ways: from the command line, or through our editor plugins.
```

Comment:
I would change this to "The Polymer Linter can be run from command line with the [Polymer CLI], or though IDE plugins, such as the [VS Code Plugin]."